src/rdkafka_aux.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * librdkafka - Apache Kafka C library
+ *
+ * Copyright (c) 2018 Magnus Edenhill
```

Comment:
You're either a good negotiator or this should be (c) Confluent. Assuming the former.